packages/web-components/src/components/promo-item/promo-item-statistic.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { html, property, customElement, LitElement } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import styles from './promo-item.scss';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * The statistic content of promo item.
+ *
+ * @element dds-promo-item-statistic
+ */
+@customElement(`${ddsPrefix}-promo-item-statistic`)
```

Comment:
Does this component need to be roped off, too?

Reply:
I'm not sure. I can wrap it in the `DDS_PROMO_GROUP` feature flag if that's something you guys would like me to do.

Reply:
sure, let's rope this one off as well.

Reply:
Done - https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/5142/commits/50ddfac6a21d282f36297004e13a68794999fd89